service/discovery/openstack/account_handle.go

```diff
@@ -0,0 +1,73 @@
+// Copyright 2024 Fraunhofer AISEC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+//           $$\                           $$\ $$\   $$\
+//           $$ |                          $$ |\__|  $$ |
+//  $$$$$$$\ $$ | $$$$$$\  $$\   $$\  $$$$$$$ |$$\ $$$$$$\    $$$$$$\   $$$$$$\
+// $$  _____|$$ |$$  __$$\ $$ |  $$ |$$  __$$ |$$ |\_$$  _|  $$  __$$\ $$  __$$\
+// $$ /      $$ |$$ /  $$ |$$ |  $$ |$$ /  $$ |$$ |  $$ |    $$ /  $$ |$$ | \__|
+// $$ |      $$ |$$ |  $$ |$$ |  $$ |$$ |  $$ |$$ |  $$ |$$\ $$ |  $$ |$$ |
+// \$$$$$$\  $$ |\$$$$$   |\$$$$$   |\$$$$$$  |$$ |  \$$$   |\$$$$$   |$$ |
+//  \_______|\__| \______/  \______/  \_______|\__|   \____/  \______/ \__|
+//
+// This file is part of Clouditor Community Edition.
+
+package openstack
+
+import (
+	"clouditor.io/clouditor/v2/api/discovery"
+	"clouditor.io/clouditor/v2/api/ontology"
+	"clouditor.io/clouditor/v2/internal/util"
+
+	"github.com/gophercloud/gophercloud/v2/openstack/identity/v3/domains"
+	"github.com/gophercloud/gophercloud/v2/openstack/identity/v3/projects"
+)
+
+// handleProject returns a [ontology.ResourceGroup] out of an existing [projects.Project].
+func (d *openstackDiscovery) handleProject(project *projects.Project) (ontology.IsResource, error) {
+	r := &ontology.ResourceGroup{
+		Id:           project.ID,
+		Name:         project.Name,
+		CreationTime: nil, // project does not have a creation date
+		GeoLocation: &ontology.GeoLocation{
+			Region: "unknown", // TODO: Can we get the region?
+		},
+		Description: project.Description,
+		Labels:      labels(util.Ref(project.Tags)),
+		ParentId:    util.Ref(project.ParentID),
+		Raw:         discovery.Raw(project),
+	}
+
+	log.Infof("Adding resource group '%s", project.Name)
```

Comment:
```suggestion
	log.Infof("Adding project '%s", project.Name)
```

Reply:
For me, projects/tenants seem to be equivalent to Azure's resource group. But then we could and should rename the function itself also